src/kOS/Communication/RunCommand.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using kOS.Safe.Compilation;
+
+namespace kOS.Communication
+{
+    public class RunCommand : InterProcCommand
+    {
+        public List<Opcode> Program;
+
+        public override void Execute(SharedObjects shared)
+        {
+            if (shared.Cpu != null)
+            {
+                shared.Cpu.RunProgram(Program);
+            }
+        }
+    }
+}
```

Comment:
You remember correctly. I've decided not to remove it in this PR. I've created a separate package for communication-related stuff and I've simply moved it there for now.